protos/core.proto

```diff
@@ -21,6 +21,7 @@ message ProjectConfig {
   string table_prefix = 11;
   
   bool use_run_cache = 10;
+  bool use_contextual_ops = 12;
```

Comment:
I find this name a bit confusing. why not `use_single_query_per_action`?

Reply:
is `use_single_task_per_action` is more correct? If we're changing task to query then I guess they're interchangeable